envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/containers/HttpsEchoContainer.kt

```diff
@@ -1,11 +1,36 @@
 package pl.allegro.tech.servicemesh.envoycontrol.config.containers
 
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import okhttp3.Response
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.ObjectAssert
 import pl.allegro.tech.servicemesh.envoycontrol.config.BaseEnvoyTest
 
-class HttpsEchoContainer : SSLGenericContainer<HttpsEchoContainer>("mendhak/http-https-echo") {
+class HttpsEchoContainer : SSLGenericContainer<HttpsEchoContainer>("mendhak/http-https-echo@$hash") {
+
+    companion object {
+        // We need to use hash because the image doesn't use tags and the tests will fail if there is an older version
+        // of the image pulled locally
+        const val hash = "sha256:6b69d5da0245157d7f9d06dfb65d0dd25fbedf5389a66d912c806572d02b0d1d"
```

Comment:
Can't you just publish a tag? ;)

Reply:
Ok, it's not your image, I've missed that.
Can't we just use latest?

Reply:
Or maybe not, I wouldn't trust some random docker image.

Reply:
if we use latest, it will use a version that is currently cached on your machine (I think so, maybe I am wrong?)

Reply:
Yea, you're right. I supposed it would be already released after almost a year of this issue being closed 🙄

Reply:
What do you mean? This version of `mendhak/http-https-echo` was relased just some hours ago :)

Reply:
https://github.com/moby/moby/issues/13331

Reply:
I see, If I understand correctly it is fixed in Docker 19.09. I have 19.03.